torch/testing/_internal/common_methods_invocations.py

```diff
@@ -158,11 +158,12 @@ def __init__(self,
                  decorators=None,  # decorators to apply to generated tests
                  safe_casts_outputs=False,  # whether op allows safe casting when writing to out arguments
                  sample_inputs_func=None,  # function to generate sample inputs
+                 sparse_sample_inputs_func=None,  # function to generate sparse sample inputs
```

Comment:
See above comment. These sparse sample inputs are used IN ADDITION to the regular sample inputs?

Reply:
Also we should clarify that setting this is what indicates the op supports sparse inputs